test/scripts/git-server-cmds/create-repo

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
```

Comment:
should we standardize that all bash scripts use

`set -euo pipefail` or at minimum `-e`

Reply:
Using set -euo pipefail now where possible, on places where we used $1 $2 ... just -eo pipefail

Reply:
cool